src/utils/sorter.js

```diff
@@ -6,6 +6,15 @@
 
 import comparator from './comparator';
 import { getBadItemError } from './error';
+import type { Comments, Tasks, AppActivityItems, FeedItems } from '../common/types/feed';
```

Comment:
Should we create `FeedItem/FeedItems` types to abstract this away from the consumers?

Reply:
Sure, though I'd prefer to avoid changing the types until a followup.